packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -2265,7 +2256,7 @@ function updateZoomTarget(viewer) {
 
   const scene = viewer.scene;
   const camera = scene.camera;
-  const zoomOptions = defaultValue(viewer._zoomOptions, {});
+  const zoomOptions = viewer._zoomOptions ?? defaultValue.EMPTY_OBJECT;
```

Comment:
The `defaultValue.EMPTY_OBJECT` is [frozen](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) and cannot be modified. Most of the places we use it are only _reading_ information from the object so this is not a problem. However some places we used `defaultValue` we do modify the properties later (like this one). There's probably discussion to be had about whether these objects _should_ be modified for later but for the sake of keeping this PR isolated let's just preserve the existing behavior.
Basically don't replace calls like these with the `EMPTY_OBJECT`, just keep the `{}`. I think this is what's causing your test failures.

Reply:
@jjspace I'll keep working on this so we can merge everything at once. Thanks for working with me! And sorry for the late response.